src/graph/rome_models.cc

```diff
@@ -642,8 +889,9 @@ static struct rcclRomeModel rome_model_81 = {
                 "N6 6 2 0 7 5 3 1 4 N4|"
                 "N7 7 3 2 6 0 4 1 5 N5|"
                 "N1 1 0 2 4 3 5 7 6 N6|"
-                "N0 0 1 2 3 4 5 6 7 N7",
-  .options    = "noCpuCheck=1,tuning=5,disableNumaMatching=1",
+		"N0 0 1 2 3 4 5 6 7 N7",
+
+  .options    = "noCpuCheck=1,tuning=5,disableNumaMatching=1,gid3=1",
```

Comment:
For code clarity, I think we should rename this option to isRoCE (or however the capitalization should be).

Reply:
Alternatively, set this to gid=3 and check that we match that value if the option is set.